example_test.go

```diff
@@ -10,6 +10,131 @@ import (
 	"inet.af/netaddr"
 )
 
+func ExampleIP() {
+	ip, err := netaddr.ParseIP("192.0.2.3")
+	if err != nil {
+		panic(err)
+	}
+
+	// netaddr.IP supports comparison using ==
+	fmt.Println(ip == netaddr.IPv4(192, 0, 2, 3))
+
+	// netaddr.IP can be used as a map key
+	hosts := make(map[netaddr.IP]string)
+	hosts[ip] = "example.net"
+	fmt.Println(hosts)
+	// Output:
+	// true
+	// map[192.0.2.3:example.net]
+}
+
+func ExampleIP_Is4() {
+	ipv4 := netaddr.MustParseIP("192.0.2.3")
+	ipv6 := netaddr.MustParseIP("2001:db8::68")
+	ipEmpty := netaddr.IP{}
+	ipv4MappedToIPv6 := netaddr.MustParseIP("::ffff:192.0.2.3")
```

Comment:
Suggest `ipMapped` or `ip4in6`, throughout.

Reply:
`ip4in6` it is